istio-prometheus-values.yaml

```diff
@@ -2,4 +2,7 @@ name: sre-service-monitoring
 session:
   secret: ciao
 prometheus_datasource:
-  url: https://prometheus.istio-system.svc:9090
\ No newline at end of file
+  name: istio-system
+  url: https://prometheus.istio-system.svc:9090
```

Comment:
the istio control plane must be a variable. In my template I called it `istio_control_plane_namespace`, so please use that one.

Reply:
This just controls the name of the datasource, not the namespace. It can be anything the user wishes to call the datasource.

Reply:
I was referring to the URL. not the name, sorry for the confusion.